CMakeLists.txt

```diff
@@ -10,6 +10,12 @@ set(CMAKE_AUTOUIC ON)
 set(CMAKE_AUTORCC ON)
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
+if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
```

Comment:
There is no "or" logical capability in cmake?

Reply:
add_compile_options?

Reply:
@DonLakeFlyer there's an or operator but for some options clang and gcc differ, so I'd prefer to keep them separated even if they add the same options for now. we don't know about the future. :)

@dagar I'll update this using add_compile_options